LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-navigation-back-cross-document-expected.txt

```diff
@@ -1,4 +1,4 @@
 
 
-FAIL navigate event for navigation.back() - cross-document assert_true: expected true got false
+PASS navigate event for navigation.back() - cross-document assert_true: expected true got false
```

Comment:
I don't understand this patch. For example, this test is clearly failing. How can we mark this as passing? Is this really  an output from the test runner?